src/Application/Migrations/24_function_ticket_reset.sql

```diff
@@ -0,0 +1,70 @@
+BEGIN;
+
+--param_handle_id is used for the log entry
+CREATE OR REPLACE FUNCTION ticket_reset(param_ticket_id bigint, param_handle_id bigint)
+  RETURNS void AS
+$$
+DECLARE
+  ticket tbl_ticket%rowtype;
+  t tbl_ticket%rowtype;
+  inital_state enum_ticket_state;
+BEGIN
+  --first get the full given ticket
+  SELECT * INTO ticket FROM tbl_ticket where id = param_ticket_id;
+
+  IF ticket IS NULL THEN
```

Comment:
As stated in https://www.postgresql.org/docs/current/static/plpgsql-statements.html#PLPGSQL-STATEMENTS-DIAGNOSTICS you can check, whether the variable FOUND is set to false.

Reply:
See paragraph starting with "The second method to determine the effects of a command..."